### DISCUSSION TITLE
rtl

### DISCUSSION THREAD

**Discussion by coffeeandwork** [beercss/beercss#289] [General]
@leonardorafael I just noticed this commit https://github.com/beercss/beercss/commit/7bd2cf9ff5c0e21bfb72bdf9023855407cb2942d#diff-a44eede226292d5a6df3a0723ae73e2d3f583d751fd41f601a70940d22600ab0

I noticed that you have replaced `padding-inline-start` with `padding-left`, `padding-inline-end` with `padding-right` and so on. While I understand that regardless of whether the UI is in rtl or ltr, it seems natural to want "left" and "right" to mean, in code, what they commonly mean in everyday language, I propose that in the context of developer experience that logic might not apply.

Most developers speak/develop in LTR languages. (JS/CSS/HTML are all LTR languages.) So a developer may be thinking/planning in an LTR context, while using `.left` and `.right`, and when they want their UIs to work in an RTL context, its a lot more intuitive if everything was flipped (as opposed to having some things flipped and other things requiring you to take extra action to flip, such as actively choosing to use `.right` instead of `.left` in RTL context, etc.)

Thoughts?

**Comment by leonardorafael** [comment]
Hi @coffeeandwork ,

I have encountered some issues with rtl implemetation on navs and dialogs, so I decided to rollback rtl on this elements. It's still a bit strange for me use left position that appear on the right position. We have some docs about rtl implementation of Material Design or other design system to follow?

**Comment by coffeeandwork** [comment]
Hi @leonardorafael,

What were the issues you encountered? Perhaps I can help debug. Looks like material design 2 has a guide on bidirectionality: https://m2.material.io/design/usability/bidirectionality.html#mirroring-layout, but I was unable to find anything for version 3 so far. I also saw this: https://rtlstyling.com/.

**Reply by leonardorafael** [reply]
I think we can follow the MD2 then, this can be another release on > v3.5.2.

**Reply by leonardorafael** [reply]
New RTL is available in [v3.5.2](https://github.com/beercss/beercss/releases/tag/v3.5.2)
